mesh/profiles/urls.py

```diff
@@ -1,9 +1,11 @@
 from django.urls import path
+
 from . import views as profile_views
-from django.http import HttpRequest
 
 urlpatterns = [
-    
-    path('biography', profile_views.bio_view),
-    path("profilePicture", profile_views.profile_picture, name="profile_picture")
+    path("biography", profile_views.bio_view),
+    path("profilePicture", profile_views.ProfilePicturesView.as_view(), name="profile_picture"),
+    path("userName", profile_views.UserNamesView.as_view(), name="user_name"),
+    path("preferredName", profile_views.PreferredNamesView.as_view(), name="preferred_name"),
+    path("preferredPronouns", profile_views.PreferredPronounsView.as_view(), name="preferred_pronouns"),
```

Comment:
This is purely a subjective ask, but can we try to use either kebab-case or snake_case in urls?